specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json

```diff
@@ -493,6 +453,96 @@
       }
     }
   },
+  "x-ms-paths": {
+    "/custom/models?op=full": {
```

Comment:
I'd use a different "fake"/uniquefying parameter - e.g. `?type=custom`

Reply:
Maybe I'm misunderstanding, but `?op=full` is the name/value for the enum. Or does that not matter here?

Reply:
In `x-ms-paths`, the part behind the ? could be anything, and has no particular value. Could be (literally) `/custom/models?This is an identifier I want it long`

That being said, I think it's a good practice to keep it to the semantic of the call, and here it's about doing `op=full`, so I would keep it with that name